### PR TITLE
Fix parts of admin-partitions guide

### PR DESCRIPTION
**Pull request by jjti**
### Description

This fixes some of the admin partitions guide. It still does not work for Kind for a couple reasons:

1. we need to use the internal IP of the K8s API (not the external LoadBalanced one):

```
# not
kubectl config view --output "jsonpath={.clusters[?(@.name=='${CLIENT_CONTEXT}')].cluster.server}"
```

2. we get TLS issues on the K8s server certs:

```txt
2023-11-14T02:56:54.305Z [ERROR] consul-server-connection-manager: ACL auth method login failed: error="rpc error: code = Unauthenticated desc = Post \"https://10.96.0.1/apis/authentication.k8s.io/v1/tokenreviews\": tls: failed to verify certificate: x509: certificate signed by unknown authority (possibly because of \"crypto/rsa: verification error\" while trying to verify candidate authority certificate \"kubernetes\")"
2023-11-14T02:56:54.305Z [ERROR] consul-server-connection-manager: connection error: error="rpc error: code = Unauthenticated desc = Post \"https://10.96.0.1/apis/authentication.k8s.io/v1/tokenreviews\": tls: failed to verify certificate: x509: certificate signed by unknown authority (possibly because of \"crypto/rsa: verification error\" while trying to verify candidate authority certificate \"kubernetes\")"
unable to start Consul server watcher: context canceled
```

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 1]
```diff
@@ -250,7 +251,7 @@ Verify that your Consul deployment meets the [Kubernetes Requirements](#kubernet
 
 1. Create a configuration for each non-default admin partition.
 
-  <CodeTabs heading="partition-workload.yaml">
```
Comment (jjti) [inline website/content/docs/enterprise/admin-partitions.mdx:253]: this snippet below uses `--values client.yaml`

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 2]
```diff
@@ -259,10 +260,10 @@ Verify that your Consul deployment meets the [Kubernetes Requirements](#kubernet
     name: consul
     enabled: false
     enableConsulNamespaces: true
-    image: hashicorp/consul-enterprise:1.14.0-ent
+    image: hashicorp/consul-enterprise:1.16.3-ent
     adminPartitions:
       enabled: true
-      name: partition-workload
```
Comment (jjti) [inline website/content/docs/enterprise/admin-partitions.mdx:265]: the screenshot uses `clients`

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 3]
```diff
@@ -310,12 +311,13 @@ You can log into the Consul UI to verify that the partitions appear as expected.
 1. If ACLs are enabled, you will need the partitions ACL token, which can be read from the Kubernetes secret. The token is an encoded string that must be decoded in base64, e.g.:
 
   ```shell-session
-  $ kubectl get secret --namespace consul ${HELM_RELEASE_SERVER}-consul-bootstrap-acl-token --template "{{ .data.token | base64decode }}"
+  $ kubectl get secret --namespace consul --context ${SERVER_CONTEXT} --template "{{ .data.token | base64decode }}" ${HELM_RELEASE_SERVER}-consul-bootstrap-acl-token
   ```
 
-  The example command gets the token using the secret name configured in the values file (`bootstrap.secretName`), decodes the secret, and prints the usable token to the console in JSON format.
```
Comment (jjti) [inline website/content/docs/enterprise/admin-partitions.mdx:316]: I think this is just out of date

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 4]
```diff
@@ -310,12 +311,13 @@ You can log into the Consul UI to verify that the partitions appear as expected.
 1. If ACLs are enabled, you will need the partitions ACL token, which can be read from the Kubernetes secret. The token is an encoded string that must be decoded in base64, e.g.:
 
   ```shell-session
-  $ kubectl get secret --namespace consul ${HELM_RELEASE_SERVER}-consul-bootstrap-acl-token --template "{{ .data.token | base64decode }}"
+  $ kubectl get secret --namespace consul --context ${SERVER_CONTEXT} --template "{{ .data.token | base64decode }}" ${HELM_RELEASE_SERVER}-consul-bootstrap-acl-token
```
Comment (jjti) [inline website/content/docs/enterprise/admin-partitions.mdx:314]: # josh @ josh-XWGWHJ0JV2 in ~/GitHub/jjti/consul-exp/telemetry-collector on git:main x [20:55:23]
$ kubectl get secret --namespace consul --context ${SERVER_CONTEXT} --template "{{ .data.token | base64decode }}" ${HELM_RELEASE_SERVER}-consul-bootstrap-acl-token
ac2e6a52-3fc1-e2b2-...

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 5]
```diff
@@ -310,12 +311,13 @@ You can log into the Consul UI to verify that the partitions appear as expected.
...
   ```
 
-  The example command gets the token using the secret name configured in the values file (`bootstrap.secretName`), decodes the secret, and prints the usable token to the console in JSON format.
+  The example command gets the secret from the default partition cluster, decodes the secret, and prints the token to the console.
+
+1. Open the Consul UI in a browser using the external IP address and port number described in a previous step (see [step 4](#get-external-ip-address)).
```
Comment (jjti) [inline website/content/docs/enterprise/admin-partitions.mdx:319]: it's step 4 not 5 now

File: website/content/docs/install/ports.mdx [hunk 6]
```diff
@@ -13,13 +13,13 @@ port.
...
 
-Before running Consul, you should ensure the following bind ports are accessible.
+Before running Consul, you should ensure the following bind ports are accessible:
 
 | Use                                                                                                           | Default Ports     |
 | ------------------------------------------------------------------------------------------------------------- | ----------------- |
 | DNS: The DNS server (TCP and UDP)                                                                             | 8600              |
 | HTTP: The HTTP API (TCP Only)                                                                                 | 8500              |
-| HTTPS: The HTTPs API                                                                                          | disabled (8501)\* |
+| HTTPS: The HTTPS API                                                                                          | disabled (8501)\* |
```
Comment (jjti) [inline website/content/docs/install/ports.mdx:22]: this has bothered me for a bit

**Comment by jjti** [comment]
Should I backport this?

File: website/content/docs/enterprise/admin-partitions.mdx [hunk 7]
```diff
@@ -310,12 +311,13 @@ You can log into the Consul UI to verify that the partitions appear as expected.
 1. If ACLs are enabled, you will need the partitions ACL token, which can be read from the Kubernetes secret. The token is an encoded string that must be decoded in base64, e.g.:
 
   ```shell-session
-  $ kubectl get secret --namespace consul ${HELM_RELEASE_SERVER}-consul-bootstrap-acl-token --template "{{ .data.token | base64decode }}"
```
Comment (jjti) [inline website/content/docs/enterprise/admin-partitions.mdx:313]: this command assumers server context

**Review by david-yu — APPROVED** [review]

**Comment by david-yu** [comment]
> Should I backport this?
Yes added backports

**Pull request merged**